### PR TITLE
Change working directory for lttng-relayd

### PR DESCRIPTION
**Pull request by PSRCode**

### CODE REVIEW

**Comment by jgalar** [comment]
Adding @eepp since he may want to look over the docs.

File: doc/man/lttng-relayd.8.txt [hunk 1]
```diff
@@ -114,6 +115,9 @@ option:-o 'PATH', option:--output='PATH'::
 See the <<output-directory,Output directory>> section above for more
 information.
 
+option:-w 'PATH', option:--working-directory='PATH'::
+    Set the working directory of lttng-relayd processes.
```
Comment (eepp) [inline doc/man/lttng-relayd.8.txt]: Let's use:

    Set the working directory of the processes this relay daemon creates.

File: doc/man/lttng-relayd.8.txt [hunk 2]
```diff
@@ -220,6 +220,11 @@ Supported on Linux only.
 +
 Search for `tcp_keepalive_intvl` in man:tcp(7) for more information.
 
+`LTTNG_RELAYD_CWD`::
```
Comment (eepp) [inline doc/man/lttng-relayd.8.txt]: Why is this named `CWD` and the equivalent option `--working-directory`? Can we have something consistent?

Also, the environment variables are alphabetically sorted in this section. You need to insert this new block at the right location, whatever the name you choose.

File: doc/man/lttng-relayd.8.txt [hunk 3]
```diff
@@ -220,6 +220,11 @@ Supported on Linux only.
...
+    Working directory for the LTTng-relayd processes.
```
Comment (eepp) [inline doc/man/lttng-relayd.8.txt]: Let's use:

    Working directory of the processes this relay daemon creates.

File: doc/man/lttng-relayd.8.txt (hunk 2 above)
Comment (PSRCode) [reply doc/man/lttng-relayd.8.txt]: After discussing with @eepp, we concluded that LTTNG_RELAYD_WORKING_DIRECTORY is a better alternative to LTTNG_RELAYD_CWD. The CWD acronym is not specific/clear enough.

File: doc/man/lttng-relayd.8.txt (hunk 3 above)
Comment (PSRCode) [reply doc/man/lttng-relayd.8.txt]: done

File: doc/man/lttng-relayd.8.txt (hunk 1 above)
Comment (PSRCode) [reply doc/man/lttng-relayd.8.txt]: done

**Comment by PSRCode** [comment]
@eepp change done. Any other comments on this pr?

**Comment by PSRCode** [comment]
Rebased on current master.

Most changes were in the test suite. Since this was done, the RELAYD_PIDS global variable with the spawned relayd id was removed. We use pgrep to find the pid.

**Comment by jgalar** [comment]
Merged, thanks!

**Pull request closed without merging**